include/config.h

```diff
@@ -30,6 +40,8 @@ struct mako_config {
 	int32_t max_visible;
 	char *output;
 	char *hidden_format;
+	enum mako_sort_criteria sort_criteria;
```

Comment:
Since this is a bitfield, this variable can take values that are not in the enum - like `MAKO_SORT_CRITERIA_TIME | MAKO_SORT_CRITERIA_URGENCY`. So this should be a `uint32` instead. Though it's a good practice to add a comment referring to the enum:

```c
uint32_t sort_criteria; // enum mako_sort_criteria
```